package.json

```diff
@@ -13,56 +13,29 @@
     "probot-app"
   ],
   "scripts": {
-    "build": "tsc",
-    "build:watch": "tsc && (tsc -w --preserveWatchOutput & nodemon)",
-    "dev": "npm run build:watch",
-    "start": "probot run ./lib/index.js",
-    "lint": "eslint 'src/**/*.ts'",
-    "lint-fix": "eslint 'src/**/*.ts' --fix",
-    "check-format": "prettier 'src/**/*.ts' -l",
-    "format": "prettier 'src/**/*.ts' --write",
-    "migrate": "knex migrate:latest --knexfile=lib/db/knexfile.js",
-    "migrate:dev": "knex migrate:latest --knexfile=src/db/knexfile.ts",
-    "test": "jest",
-    "test:watch": "jest --watch --notify --notifyMode=change --coverage"
+    "bootstrap": "lerna link && lerna bootstrap",
```

Comment:
`lerna link` should not be needed, `lerna bootstrap` will also link the packages.